templates/bundles/DemosPlanCoreBundle/DemosPlanStatement/includes/new_public_participation_statement_confirm_content.html.twig

```diff
@@ -0,0 +1,171 @@
+<h3>{{ "mydata"|trans }}</h3>
+
+<dl class="{{ 'layout__item layout--flush u-1-of-3 u-pl-0 u-1-of-1-lap-down'|prefixClass }}">
+
+    <dt class="{{ 'layout__item u-1-of-4 u-mb u-mr'|prefixClass }}">{{ "name"|trans }}</dt>
+
+    <dd class="{{ 'layout__item u-7-of-12 u-ml'|prefixClass }}">{{ templateVars.user.firstname|default("---") }} {{ templateVars.user.lastname|default("----") }}</dd>
+
+    <dt class="{{ 'layout__item u-1-of-4 u-mb u-mr'|prefixClass }}">{{ "email"|trans }}</dt>
+
+    <dd class="{{ 'layout__item u-7-of-12 u-ml'|prefixClass }}">{{ templateVars.user.email|default("----") }}</dd>
+
+    {# until the user has no address in our system, we dont want him to see unneccessary information #}
+    {# in gateway mode the address comes from there #}
+    {% if projectType == 'gateway' %}
+
+        <dt class="{{ 'layout__item u-1-of-4 u-mb u-mr'|prefixClass }}">{{ "street"|trans }}</dt>
+
+        <dd class="{{ 'layout__item u-7-of-12 u-ml'|prefixClass }}">{{ templateVars.user.address.street|default("----") }}</dd>
+
+        <dt class="{{ 'layout__item u-1-of-4 u-mb u-mr'|prefixClass }}">{{ "street.number"|trans }}</dt>
+
+        <dd class="{{ 'layout__item u-7-of-12 u-ml'|prefixClass }}">{{ templateVars.user.address.houseNumber|default("----") }}</dd>
+
+        <dt class="{{ 'layout__item u-1-of-4 u-mb u-mr'|prefixClass }}">{{ "postalcode"|trans }}</dt>
+
+        <dd class="{{ 'layout__item u-7-of-12 u-ml'|prefixClass }}">{{ templateVars.user.address.postalcode|default("----") }}</dd>
+
+        <dt class="{{ 'layout__item u-1-of-4 u-mb u-mr'|prefixClass }}">{{ "city.name"|trans }}</dt>
+
+        <dd class="{{ 'layout__item u-7-of-12 u-ml'|prefixClass }}">{{ templateVars.user.address.city|default("----") }}</dd>
+    {% endif  %}
+
+</dl>
+
+<div class="{{ 'layout__item u-2-of-3 u-1-of-1-lap-down u-pl-0-lap-down u-mb-0_5-lap-down'|prefixClass }}">
+    {% block vote_data %}
```

Comment:
I feel that `vote_data` is not the ideal name. How about `user_data_hint`?

Reply:
Good point, I agree. Done in [bcd9cec](https://github.com/demos-europe/demosplan-core/pull/1780/commits/bcd9ceca8a5e2d3d9ce0724afd947f452a0a3f32)